src/main/java/bio/terra/cli/service/SamService.java

```diff
@@ -547,16 +547,16 @@ private void handleClientExceptions(
             apiEx.getMessage());
 
         // try to deserialize the response body into an ErrorReport
-        String apiExMsg;
-        try {
-          ErrorReport errorReport =
-              JacksonMapper.getMapper().readValue(apiEx.getResponseBody(), ErrorReport.class);
-          apiExMsg = errorReport.getMessage();
-        } catch (JsonProcessingException jsonEx) {
-          logger.debug(
-              "Error deserializing SAM exception ErrorReport: {}", apiEx.getResponseBody());
-          apiExMsg = apiEx.getResponseBody();
-        }
+        String apiExMsg = apiEx.getResponseBody();
+        if (apiExMsg != null)
+          try {
+            ErrorReport errorReport =
+                JacksonMapper.getMapper().readValue(apiEx.getResponseBody(), ErrorReport.class);
+            apiExMsg = errorReport.getMessage();
+          } catch (JsonProcessingException jsonEx) {
+            logger.debug(
```

Comment:
why is this only a debug message? Is it common to have Sam exceptions that can't be parsed?

Reply:
I don't know about common, but I have seen it before. This function is just error handling, and I don't want e.g. a NPE to obscure the actual exception that SAM threw.